docs/kyma/docs/034-inst-install-subcomponents.md

```diff
@@ -14,14 +14,14 @@ To specify whether to install a given core subcomponent on Minikube, use the `ma
 
 Example:
 
-To enable the Azure Broker subcomponent, run the following command:
+To disable the API-Controller subcomponent, run the following command:
```

Comment:
todo: test this

Reply:
why do you describe PAI controller in this PR?

Reply:
discussed. it was used as the example